From a6dc42deaea6cb9e148e4d93a5aae0e5ebc599d1 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Mon, 29 Jul 2013 15:44:10 -0400
Subject: [PATCH 130/225] Ticket 47447 - logconv.pl man page missing
 -m,-M,-B,-D

Fix Description:  Added the missing info, and also updated the main arguments
                  with their long option names.

https://fedorahosted.org/389/ticket/47447

Reviewed by: nhosoi(Thanks!)
(cherry picked from commit 4ea4c8bf91f5c28bd643b0c06f590ec164156358)
(cherry picked from commit ed245c45813255400c6962707f7269caa253e953)
(cherry picked from commit 65eb63b8a35afe5026386b1dda6eb321f52e18f2)
(cherry picked from commit 557e95b06c513b5234aaebffb0f82ba6da5d0ff6)
---
 man/man1/logconv.pl.1 | 36 ++++++++++++++++++++++++++----------
 1 file changed, 26 insertions(+), 10 deletions(-)

diff --git a/man/man1/logconv.pl.1 b/man/man1/logconv.pl.1
index ab81c69..c2a8cd8 100644
--- a/man/man1/logconv.pl.1
+++ b/man/man1/logconv.pl.1
@@ -29,30 +29,46 @@ line
 .SH OPTIONS
 A summary of options is included below:
 .TP
-.B \fB\-h\fR 
+.B \fB\-h, \-\-help\fR 
 help/usage
 .TP
-.B \fB\-d\fR <Directory Managers DN>
+.B \fB\-d, \-\-rootDN\fR <Directory Managers DN>
 DEFAULT \-> cn=directory manager
 .TP
-.B \fB\-s\fR <Number of results to return per category>
+.B \fB\-D, \-\-data\fR <Location for temporary data files>
+DEFAULT \-> /tmp
+.br
+TIP \-> If there is not enough RAM, a RAM disk can be used instead:
+
+       mkdir /dev/shm/logconv, and use this directory for the "-D" value.
+.TP
+.B \fB\-s, \-\-sizeLimit\fR <Number of results to return per category>
 DEFAULT \-> 20
 .TP
-.B \fB\-X\fR <IP address to exclude from connection stats>
+.B \fB\-X, \-\-excludeIP\fR <IP address to exclude from connection stats>
 E.g. Load balancers
 .TP
-.B \fB\-v\fR show version of tool
+.B \fB\-v, \-\-version\fR show version of tool
 Print version of the tool
 .TP
-.B \fB\-S\fR <time to begin analyzing logfile from>
+.B \fB\-S, \-\-startTime\fR <time to begin analyzing logfile from>
 Time to begin analyzing logfile from
-E.g. [28/Mar/2002:13:14:22 \fB\-0800]\fR
+E.g. [28/Mar/2002:13:14:22 \f \-0800]\fR
 .TP
-.B \fB\-E\fR <time to stop analyzing logfile>
+.B \fB\-E, \-\-endTime\fR <time to stop analyzing logfile>
 Time to stop analyzing logfile from
-E.g. [28/Mar/2002:13:24:62 \fB\-0800]\fR
+E.g. [28/Mar/2002:13:24:62 \f \-0800]\fR
+.TP
+.B \fB\-M, \-\-reportFileMins\fR <CSV output file>
+This option creates a CSV report for spreadsheets.
+.TP
+.B \fB\-m, \-\-reportFileSecs\fR <CSV output file>
+This option creates a CSV report for spreadsheets.
+.TP
+.B \fB\-B, \-\-bind\fR <ALL | ANONYMOUS | "Bind DN">
+This generates a report based on either ALL bind dn's, anonymous binds, or a specific DN.
 .TP
-\fB\-V\fR <enable verbose output \- includes all stats listed below>
+\fB\-V, \-\-verbose\fR <enable verbose output \- includes all stats listed below>
 Verbose output
 .TP
 .B \fB\-[efcibaltnxgju]\fR
-- 
1.8.1.4

